feat(config): added eslint-config-angular-ts

Erbenos opened the pull request:
Please do further specify the RxJS rules as you see fit.

Erbenos commented:
Do we want to force explicit accessibility on getter and setters as well?

fvoska commented:
> Do we want to force explicit accessibility on getter and setters as well?

Yep

Erbenos commented:
That's included then... only constructors are the exception. Asked cause other project didn't have the explicit accessibility on getters/setters.

Erbenos commented:
Added the prefer-readonly rule, though it seems I can't force it to public props, see  https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-readonly.md

Changed RxJS config so that the recommended preset from https://github.com/cartant/eslint-plugin-rxjs/blob/main/source/configs/recommended.ts gets used.

Updated the example app.
